<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>413</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuPineboo">
    <property name="title">
     <string>Pineboo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="configMenu">
    <property name="title">
     <string>Configuración</string>
    </property>
    <addaction name="actionTipografia"/>
   </widget>
   <widget class="QMenu" name="menuVentanas">
    <property name="title">
     <string>Ventana</string>
    </property>
    <widget class="QMenu" name="menuVistas">
     <property name="title">
      <string>Vistas</string>
     </property>
     <addaction name="actionFavoritos"/>
     <addaction name="actionModulos"/>
     <addaction name="separator"/>
     <addaction name="actionAlinear"/>
    </widget>
    <addaction name="menuVistas"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcercaPineboo"/>
    <addaction name="actionAcercaQt"/>
   </widget>
   <addaction name="menuPineboo"/>
   <addaction name="configMenu"/>
   <addaction name="menuVentanas"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>569</y>
     <width>20</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <action name="actionSalir">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionEstilo">
   <property name="text">
    <string>Estilo</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionTipografia">
   <property name="text">
    <string>Tipografía</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionAcercaPineboo">
   <property name="text">
    <string>Acerca  de Pineboo</string>
   </property>
  </action>
  <action name="actionAcercaQt">
   <property name="text">
    <string>Acerca de Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionVistas">
   <property name="text">
    <string>Vistas</string>
   </property>
  </action>
  <action name="actionFavoritos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Favoritos</string>
   </property>
  </action>
  <action name="actionModulos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Modulos</string>
   </property>
  </action>
  <action name="actionAlinear">
   <property name="text">
    <string>Alinear</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionSalir</sender>
   <signal>toggled(bool)</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
